data/campaigns/Winds_of_Fate/scenarios/05x_Unexpected_Arrival.cfg

```diff
@@ -0,0 +1,329 @@
+        [message]
+            speaker=Reshan
+            image_pos=right
+            mirror=yes
+            message= _ "They are ‘Saurians’."
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Their likeness to hatchlings is disarming. Disturbingly so. I will not have them near my fli--"
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "Enough."
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "I shall have a closer survey of the greatland focused on game abundance.
+
+Reshan, see to it at once."
+        [/message]
+        [message]
+            speaker=Reshan
+            image_pos=right
+            mirror=yes
+            message= _ "By your will."
+        [/message]
+        [move_unit]
+            id=Reshan
+            to_x,to_y=33,4
+        [/move_unit]
+        [put_to_recall_list]
+            id=Reshan
+        [/put_to_recall_list]
+        [message]
+            speaker=Karron
+            message= _ "Gorlack, arriving here, I noticed a large gathering of game to the northeast.
+A nest of haldric."
+        [/message]
+        [message]
+            speaker=Zedrix
+            image_pos=right
+            mirror=yes
+            message= _ "You ssaw Demon King?! You ssssaw <b>The Haldric</b>?"
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Gorlack, your hatchling-things cry nonsense. You must gain control over them."
+        [/message]
+        [message]
+            speaker=Arinexis
+            image_pos=right
+            mirror=yes
+            message= _ "Not nonsensse! Creaturess you sssaw call themsselvesss “Humanss”. Lumbering, brutish, odiouss monstersss covered in moisst sskin of toad. Hirelingsss sssummoned from underssside of world by elvesss of deep, dark foresst-- Wesssmere foressst.
+
+Thesse humanss ssslaughtered all but my entire clan! Upon the ruinsss of my hatchplace they raissed their invulnerable fortress of Elensssfar. Pray dragonsss, rain jussst fire down upon thesse abominationsss!"
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Gorlack, we should harvest them.
+There is meat enough there to feed our three flights a full year."
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "Reshan has counseled against it until more is known of them.
+The Glider Memory often speaks of underestimated prey turning to become the hunter.
+Also that this greatland hides terrible danger."
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "You speak with Reshan’s fearful tongue.
+I remember, long ago...
+
+You would speak of our great battles to be won over the Rogues.
```

Comment:
The is the only instance that I noticed where 'rogue' is capitalised. Unless it's established somewhere that these rebel drakes are formally titled Rogues, I think this should be lower-case as well.

Reply:
Switched "rogue" to lower case except where an official invocation of the "Ways of Morogor" is being made (and put a description of this grammar rule in the "Morogor_Dialect" document.